website/marietje/templates/marietje/authorize.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'tosti/base.html' %}
```

Comment:
Do we want this to be a front end page? I think a admin page might be nicer / make more sense

Reply:
I’m not sure how you would see this as an admin page, also I really like how the page looks like this. Maybe we can adjust the header so that it does not seem like a “normal” page?

Reply:
I took a look at how to do this, this is really difficult so if you agree we can create an issue for this and do this at a later stage

Reply:
wouldn't it just work if you would extend the admin base template?

Reply:
if not, fine by me